bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -135,6 +154,37 @@ public String getSrcset() {
                 // in case of dm image and auto smartcrop the srcset needs to generated client side
                 if (dmImage && StringUtils.equals(smartCropRendition, SMART_CROP_AUTO)) {
                     srcSet = EMPTY_PIXEL;
+                } else if (ngdmImage && StringUtils.equals(smartCropRendition, SMART_CROP_AUTO)) {
+                    String endPointUrl = "https://" + nextGenDynamicMediaConfig.getRepositoryId() + metadataDeliveryEndpoint;
+                    HttpGet get = new HttpGet(endPointUrl);
+                    get.setHeader("X-Adobe-Accept-Experimental", "1");
+                    try {
+                        HttpResponse response = client.execute(get);
+                        StatusLine statusLine = response.getStatusLine();
+                        int statusCode = statusLine.getStatusCode();
+                        if (statusCode == HttpStatus.SC_OK) {
+                            HttpEntity e = response.getEntity();
```

Comment:
You should always close the response. Better use the `execute` method taking a `ResponseHandler` (https://hc.apache.org/httpcomponents-client-4.5.x/current/httpclient/apidocs/org/apache/http/client/HttpClient.html#execute(org.apache.http.client.methods.HttpUriRequest,%20org.apache.http.client.ResponseHandler)) and directly act on the returned input stream.

Reply:
@kwin Thanks. Incorporated the feedback in new commit.